crates/fund/examples/fund.rs

```diff
@@ -42,10 +42,13 @@ async fn fund_everything(faucet: &str, coordinator: &str) -> Result<()> {
 
     let bitcoind = bitcoind::Bitcoind::new(client, faucet.to_string() + "/bitcoin");
 
-    bitcoind
-        .fund(&coord_addr, Amount::ONE_BTC)
-        .await
-        .context("Could not fund the faucet's on-chain wallet")?;
+    for _ in 0..5 {
```

Comment:
🤔 I think the `generatetoaddress` api that is called for funding accepts a count argument.

Reply:
I tried it but it didn't have the expected result and then I gave up because this has the same effect